static/scripts/topbar.js

```diff
@@ -0,0 +1,28 @@
+$(document).ready(() => {
+	let sessionTokenExpirationDate;
+
+	const route = '/api/v3/oauth/session-token/expiration';
+	$.getJSON(route)
+		.done((response) => {
+			sessionTokenExpirationDate = new Date(response.expiresAt);
+		});
+
+	$('.user-menu-btn').click(() => {
+		const isMenuClosing = $('.user-menu').hasClass('open');
+		if (isMenuClosing) {
+			return;
+		}
+
+		if (!sessionTokenExpirationDate) {
+			$('#external-logout').addClass('disabled');
```

Comment:
Also set the `tabindex="-1"` so you cannot use keyboard navigation to access the link and add `aria-disabled="true"` for accessibility